src/parser.js

```diff
@@ -0,0 +1,40 @@
+const bent = require('bent');
+const jsdom = require('jsdom');
+
+const request = bent('string');
+const url = 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List';
+const { JSDOM } = jsdom;
+const nameTest = /^name/i;
```

Comment:
Move all these variables into the scope in which they are defined.  No need for module level variables here.